front-end/src/themes/theme.ts

```diff
@@ -1,4 +1,16 @@
 export const theme = {
+	border: {
```

Comment:
I think there's a misunderstanding. I only meant to keep the color as a variable (because it's reused eventually)
but I didn't mean to use a variable for every border property, our theme will unnecessarily grow too big.

and then on the component use:
```css
border-style: solid;
border-color: red_primary;
border-width: 0.1rem;
```

Reply:
ah yeah, that makes more sense